src/tailwind.css

```diff
@@ -30,6 +30,12 @@
   a:hover {
     @apply text-cerulean;
   }
+
+  button.button-icon svg {
+    max-width: 16px;
+    margin-left: auto;
+    margin-right: auto;
+  }
```

Comment:
```suggestion
  .button-icon svg {
    max-width: 16px;
    margin-left: auto;
    margin-right: auto;
  }
```

this would have to be repeated at application end, um mmm